CHANGELOG.md

```diff
@@ -10,28 +10,26 @@
 
 #### V3
 
-- Deployed V3 Vault to Mainnet, Gnosis, Arbitrum, Base, Avalanche and Sepolia.
-- Deployed V3 Weighted Pool to Mainnet, Gnosis, Arbitrum, Base, Avalanche and Sepolia.
-- Deployed V3 Stable Pool to Mainnet, Gnosis, Arbitrum, Base, Avalanche and Sepolia.
-- Deployed V3 Router to Mainnet, Gnosis, Arbitrum, Base, Avalanche and Sepolia.
-- Deployed V3 Batch Router to Mainnet, Gnosis, Arbitrum, Base, Avalanche and Sepolia.
-- Deployed V3 Buffer Router to Mainnet, Gnosis, Arbitrum, Base, Avalanche and Sepolia.
-- Deployed V3 Vault Explorer to Mainnet, Gnosis, Arbitrum, Base, Avalanche and Sepolia.
+- Deployed V3 Vault to all networks.
+- Deployed V3 Weighted Pool to all networks.
+- Deployed V3 Batch Router to all networks.
+- Deployed V3 Buffer Router to all networks.
+- Deployed V3 Vault Explorer to all networks.
 - Deployed V3 Stable Surge Pool Factory to Mainnet, Gnosis, Arbitrum, Base and Sepolia.
-- Deployed V3 Composite Liquidity Router (V2) to Mainnet, Gnosis, Arbitrum, Base, Avalanche and Sepolia.
-- Deployed V3 Balancer Contract Registry to Mainnet, Gnosis, Arbitrum, Base, Avalanche and Sepolia.
-- Deployed V3 Aggregator Router to Mainnet, Gnosis, Arbitrum, Base, Avalanche and Sepolia.
-- Deployed V3 MEV Capture Hook to Mainnet, Gnosis, Arbitrum, Base and Sepolia.
-- Deployed V3 Protocol Fee Sweeper to Mainnet, Gnosis, Arbitrum, Base, Avalanche and Sepolia.
-- Deployed V3 Router (V2) to Mainnet, Gnosis, Arbitrum, Base, Avalanche and Sepolia.
+- Deployed V3 Composite Liquidity Router (V2) to all networks.
+- Deployed V3 Balancer Contract Registry to all networks.
+- Deployed V3 Aggregator Router to all networks.
+- Deployed V3 MEV Capture Hook to Mainnet, Gnosis, Arbitrum, Base, Optimism and Sepolia.
+- Deployed V3 Protocol Fee Sweeper to all networks.
+- Deployed V3 Router (V2) to all networks.
 - Deployed V3 Cow Swap Fee Burner to Mainnet, Gnosis, Arbitrum, Base and Sepolia.
-- Deployed V3 Liquidity Bootstrapping Pool to Mainnet, Gnosis, Arbitrum, Base, Avalanche and Sepolia.
+- Deployed V3 Liquidity Bootstrapping Pool to all networks.
 - Deployed V3 Hook Examples to Sepolia.
-- Deployed V3 Protocol Fee Controller V2 to Mainnet, Gnosis, Arbitrum, Base, Avalanche and Sepolia.
+- Deployed V3 Protocol Fee Controller V2 to all networks.
 - Deployed V3 Protocol Fee Controller Migration to Mainnet, Gnosis, Arbitrum, Base and Sepolia.
 - Deployed V3 Balancer Contract Registry Initializer to Mainnet, Gnosis, Arbitrum, Base and Sepolia.
```

Comment:
Don't we need the registry initializer here, too? We could redeploy it to initialize with the stable pool v2, which we didn't have when the current version was deployed.

Reply:
let's wait a bit. We'll also deploy stable surge v2 and 2-clps soon